static/login.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset=utf-8>
+    <meta name=viewport content="initial-scale=1, minimum-scale=1, width=device-width">
+    <link rel="shortcut icon" href="/favicon.png">
+    <script src="//www.gstatic.com/firebasejs/ui/live/0.4/firebase-ui-auth.js"></script>
+    <link type="text/css" rel="stylesheet" href="//www.gstatic.com/firebasejs/ui/live/0.4/firebase-ui-auth.css" />
+    <link href="/css/login.css" rel="stylesheet" type="text/css">
+    <title>Copilot Volunteer Login</title>
+  </head>
+  <body>
+    <a href=//www.copilot.help/><h1 id=logo aria-label=Copilot>Copilot</h1></a>
+    <p>Volunteers are the core of our team. This is your chance to help students in need. <ins>This website lets you anonymously support peers who are going through tough times and make sure they get the help they need. <a href="mailto:team@copilot.help">Contact us</a> for an invitation to our growing team.</p>
+
+    <p>Welcome to the team. <ins>Sign in with Google to continue.</ins></p>
+    <div id="login" class="loginBtn google">Login with Google</div>
```

Comment:
Make this button span the size of the text, not the width of the container. I think that would look better.